src/Whoops/Exception/Formatter.php

```diff
@@ -47,8 +47,20 @@ public static function formatExceptionAsDataArray(Inspector $inspector, $shouldA
         return $response;
     }
 
+    /**
+     * @param Inspector $inspector
+     *
+     * @return string
+     *
+     * @deprecated
+     */
     public static function formatExceptionPlain(Inspector $inspector)
     {
+        @trigger_error(
+            'Using plain exception formatter is deprecated and will be removed in next releases',
```

Comment:
The msg should mention what to use instead

Reply:
Done, additionally extended the test for deprecation error triggering